ios-sdkTests/MapViewControllerTests.swift

```diff
@@ -170,27 +170,35 @@ class MapViewControllerTests: XCTestCase {
   }
   
   func testLoadSceneFile() {
-    controller.loadSceneFile("path")
+    try? controller.loadSceneFile("path")
     XCTAssertEqual(tgViewController.scenePath, "path")
   }
   
   func testLoadSceneFileWithUpdates() {
-    let updates = [TGSceneUpdate]()
-    controller.loadSceneFile("path", sceneUpdates: updates)
+    let update = TGSceneUpdate(path: "path", value: "value")
+    var updates = [TGSceneUpdate]()
+    updates.append(update)
+    try? controller.loadSceneFile("path", sceneUpdates: updates)
     XCTAssertEqual(tgViewController.scenePath, "path")
-    XCTAssertEqual(tgViewController.sceneUpdates, updates)
+    XCTAssertEqual(tgViewController.sceneUpdates.count, 2)
+    XCTAssertTrue(tgViewController.sceneUpdates.contains(update))
+    XCTAssertEqual(tgViewController.sceneUpdates.last?.path, "global.sdk_mapzen_api_key")
   }
   
   func testLoadSceneFileAsync() {
-    controller.loadSceneFileAsync("path" , onSceneLoaded: nil)
+    try? controller.loadSceneFileAsync("path" , onSceneLoaded: nil)
```

Comment:
I guess this is an OK test? Really I think a more interesting test could be to use Xcode's asynchronous testing framework to guarantee that the load happens, but that would require a view hierarchy in place so probably we should eventually work towards some integration testing for this.